src/components/Assets/UploadAssetsComponent.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Progress, Icon } from 'semantic-ui-react';
+import Dropzone from 'react-dropzone';
+import '../../_css/UploadAssets.css';
+
+const StatusMessage = props => <div className={`${props.className} `}>{props.message}</div>;
+
+const uploadStatus = (success, error) => {
```

Comment:
This should be a component in its own file. I'm having trouble reading to understand the code due to the way the file is structured

Reply:
You haven't actioned [this feedback](https://github.com/AndelaOSP/art-dashboard/pull/177/files#r239451074)

Reply:
Working on it.